app/views/podcast_episodes/show.html.erb

```diff
@@ -49,7 +49,7 @@
         </time>
       <% end %>
     </div>
-    <div id="record-<%= @episode.slug %>" class="record-wrapper" data-podcast="<%= @podcast.slug %>" data-episode="<%= @episode.slug %>">
+    <div id="record-<%= @episode.slug %>" class="record-wrapper" data-podcast="<%= @podcast.slug %>" data-episode="<%= @episode.slug %>" tabindex="0" aria-pressed="false">
```

Comment:
Refactor (blocking): Instead of adding a `tabIndex` attribute to a `<div />`, use a button, i.e. prefer a semantic HTML element. We would most likely also need an `aria-label` attribute for screen readers. Some styling will need to be done most likely to retain the original look.